javalib/src/main/scala/java/lang/process/GenericProcess.scala

```diff
@@ -78,6 +78,9 @@ private[process] abstract class GenericProcess(val handle: GenericProcessHandle)
 // Represents ProcessHandle for process started by Scala Native runtime
 // Cannot be used with processes started by other programs
 private[process] abstract class GenericProcessHandle extends ProcessHandle {
+  override final def pid(): Long = ipid
+
+  val ipid: Int
```

Comment:
Correct as is.

I offer the suggestion that a two line comment stating something like the below would help people who wonder
what an 'ipid' is and trace the definition back to its first
occurrence.

// read as 'internal pid' (or as 'implementation pid' or as 'Int pid'?)
// All supported architectures use an at most 32 bit process id 

I am not suggesting a name change at each use site